SSA/Experimental/Bits/Fast/ForLean.lean

```diff
@@ -0,0 +1,28 @@
+import Mathlib.Algebra.Group.Defs
```

Comment:
Can we move this into SSA/Projects/InstCombine/ForLean.lean instead?

Reply:
Yes, that sounds like a good idea.

Reply:
Also, do we need these mathlib includes?